README.md

```diff
@@ -169,8 +169,48 @@ bazel build --enable_runfiles build_pip_pkg
 TF_NEED_CUDA=1 bazel-bin/build_pip_pkg artifacts
 pip install artifacts/tensorflow_recommenders_addons_gpu-*.whl
 ```
+##### Apple Silicon Support
```

Comment:
I recommend adding 'Beta release' here, but you make the decision.
##### Apple Silicon Support (Beta Release)

Reply:
Added "(Beta Release)".